core/cloudflow-akka-tests/src/test/scala/cloudflow/akkastream/scaladsl/AkkaStreamletConsumerGroupSpec.scala

```diff
@@ -154,18 +154,18 @@ class AkkaStreamletConsumerGroupSpec extends EmbeddedKafkaSpec(kafkaPort, zkPort
 
     def run(testData: List[Data], outlet: String): StreamletExecution = {
       val gen     = new Generator(testData)
-      val context = new AkkaStreamletContextImpl(definition(StreamletRef, outlet), system)
+      val context = new AkkaStreamletContextImpl(definition(outlet), system)
       gen.setContext(context)
       gen.run(context)
     }
 
-    def definition(streamletRef: String, outlet: String) = StreamletDefinition(
+    def definition(outlet: String) = StreamletDefinition(
```

Comment:
(minor/cleanup) I think that the concept of an abstract 'outlet' does not exist anymore. I think it would be more clear to call this "topic"